docs2/site/docs/migrations/migration5.md

```diff
@@ -13,42 +13,43 @@ It skips adding a mapping for any graph type marked with the `[DoNotMapClrType]`
 In v5, it will also skip adding the mapping if the CLR type is marked with the
 `[DoNotMapClrType]` attribute.
 
-### Input Extensions support
+### 2. Input Extensions support
 
 `Extensions` deserialized from GraphQL requests can now be set on the `ExecutionOptions.Extensions` property
 and passed through to field resolvers via `IResolveFieldContext.InputExtensions`. Note that standard .NET
-dictionaries (such as `Dictionary<TKey, TValue>`) are thread-safe for read-only operations.
+dictionaries (such as `Dictionary<TKey, TValue>`) are thread-safe for read-only operations. Also you can
+access these extensions from validation rules via `ValidationContext.Extensions`.
 
 ## Breaking Changes
 
-### UnhandledExceptionDelegate
+### 1. UnhandledExceptionDelegate
 
 `ExecutionOptions.UnhandledExceptionDelegate` and `IExecutionContext.UnhandledExceptionDelegate`
 properties type was changed from `Action<UnhandledExceptionContext>` to `Func<UnhandledExceptionContext, Task>`
 so now you may use async/await for exception handling. In this regard, some methods in `ExecutionStrategy` were
 renamed to have `Async` suffix.
 
-### `IDocumentCache` now has asynchronous methods instead of synchronous methods.
+### 2. `IDocumentCache` now has asynchronous methods instead of synchronous methods.
```

Comment:
@Shane32 Are you fine with such numbering? It seemed to me that the numbering (in any form) better shows the amount of changes and allows to navigate among them.

Reply:
Sure